server/models/User.js

```diff
@@ -19,7 +19,13 @@ const userSchema = new mongoose.Schema({
   register_date: {
     type: Date,
     default: Date.now
-  }
+  },
+  profileId: [
```

Comment:
Question: Since the profileId field here is an array of object ids, does this mean that a user can have multiple profiles?

Reply:
Thank you for your reminder. I prefer a user who has one profile only. How do you think?

Reply:
Yes, having only one profile for a user seems more appropriate for the scope of our project 👍🏽

Reply:
Thanks. Done